projects/demo/src/modules/directives/dropdown-hover/examples/5/index.html

```diff
@@ -14,7 +14,7 @@
     <tui-data-list>
         <a
             href="https://github.com/taiga-family/taiga-ui"
-            iconStart="assets/images/github.svg"
+            iconStart="assets/icons/github.svg"
```

Comment:
btw

https://lucide.dev/icons/?focus=&search=github

Reply:
I know, but they don't have discord or telegram so I still use it here for uniformity